app/logic/daily_digest.rb

```diff
@@ -0,0 +1,32 @@
+class DailyDigest
+
+  def send
+    params = get_statistic
+    daily_statistic = DailyProgressDigest.new params
+
+    Notifier.digest(daily_statistic).deliver
+  end
+
+  private
+
+  def get_statistic
+    recipients = get_recipients
+    completed = completed_amount
+    remaining = incompleted_amount
+
+    Hash(recipients: recipients, completed: completed, remaining: remaining)
+  end
+
+  def get_recipients
+    User.all.map {|user| user.email}
```

Comment:
Btw, FYI: https://github.com/bbatsov/ruby-style-guide#single-action-blocks. Usage of `map(&:email)`